src/screens/SosContactForm/index.js

```diff
@@ -1,24 +1,146 @@
-import React from 'react';
-import {View, Button, StyleSheet, Text, TextInput} from 'react-native';
+import React, { useState } from 'react';
+
+import {
+  View,
+  Text,
+  TextInput,
+  Button,
+  TouchableOpacity,
+  StyleSheet,
+} from 'react-native';
+
+import appApiClient from '../../api/appApiClient';
+
+export default function EmergencyScreen({ navigation }) {
+  const initialContactState = {
+    name: '',
+    number: '',
```

Comment:
not sure if you can call variable number, may be it is reserved word in javascript